solution/src/main/openapi/solution.yaml

```diff
@@ -749,6 +749,10 @@ components:
         repository:
           type: string
           description: the registry repository containing the image
+        alwaysPull:
+          type: boolean
+          description: the image pull policy
```

Comment:
I guess the description is not really accurate. It doesn't actually store the pull policy.